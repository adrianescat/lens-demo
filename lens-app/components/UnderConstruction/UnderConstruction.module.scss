.underConstruction {
  display: flex;
  flex-direction: column;
  font-family: "MSQTSemiBold";
  padding: 20px;
  justify-content: center;
  align-items: center;
  width: 100%;
}

@media screen and (min-width: 930px) {
  .underConstruction {
    width: 80%;
    min-height: 90vh;
  }
}