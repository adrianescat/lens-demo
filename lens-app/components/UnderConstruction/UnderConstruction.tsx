import React, { ReactElement } from 'react'
import Image from 'next/image';
import styles from './UnderConstruction.module.scss';

export default function UnderConstruction(): ReactElement {
  return (
    <div className={styles.underConstruction}>
      <Image
        src="/img/under_construction.svg"
        alt="Under construction"
        width={300}
        height={300}
      />

      <div>Coming Soon</div>
    </div>
  )
}
