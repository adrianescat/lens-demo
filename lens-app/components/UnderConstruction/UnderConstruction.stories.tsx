import React from 'react';
import UnderConstruction from '.';

export default {
  component: UnderConstruction,
  title: 'UnderConstruction',
};

export function Basic(): React.ReactNode {
  return (
    <UnderConstruction />
  );
}
