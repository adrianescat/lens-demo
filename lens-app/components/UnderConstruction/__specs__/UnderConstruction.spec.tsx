import React from 'react';
import { shallow } from 'enzyme';
import UnderConstruction from '../UnderConstruction';

let wrapper: any;
let props: any;

beforeEach(() => {
  props = {};
  wrapper = shallow(<UnderConstruction {...props} />);
});

test('should match snapshots', () => {
  expect(wrapper).toMatchSnapshot();
});