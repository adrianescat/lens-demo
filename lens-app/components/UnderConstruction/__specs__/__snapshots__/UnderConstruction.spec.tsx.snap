// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshots 1`] = `
<div
  className="underConstruction"
>
  <Image
    alt="Under construction"
    height={300}
    src="/img/under_construction.svg"
    width={300}
  />
  <div>
    Coming Soon
  </div>
</div>
`;
